CorsixTH/Lua/entities/humanoids/patient.lua

```diff
@@ -962,7 +990,7 @@ end
 function Patient:updateDynamicInfo()
   local action_string = self.action_string or ""
   local info = ""
-  if self.going_home then
+  if self.going_home or self.going_die then
```

Comment:
self.going_die how is this different than going_to_die, why such similar names. Easily confused.